sentry-graphql/src/main/java/io/sentry/graphql/SentryDataFetcherExceptionHandler.java

```diff
@@ -36,13 +39,29 @@ public SentryDataFetcherExceptionHandler(final @NotNull DataFetcherExceptionHand
   }
 
   @Override
-  @SuppressWarnings("deprecation")
-  public DataFetcherExceptionHandlerResult onException(
-      final @NotNull DataFetcherExceptionHandlerParameters handlerParameters) {
+  public CompletableFuture<DataFetcherExceptionHandlerResult> handleException(
+      DataFetcherExceptionHandlerParameters handlerParameters) {
     final Hint hint = new Hint();
     hint.set(GRAPHQL_HANDLER_PARAMETERS, handlerParameters);
 
     hub.captureException(handlerParameters.getException(), hint);
-    return delegate.onException(handlerParameters);
+    return delegate.handleException(handlerParameters);
+  }
+
+  @SuppressWarnings("deprecation")
+  public @Nullable DataFetcherExceptionHandlerResult onException(
```

Comment:
@adinauer this was not nullable before. Should we rather return an empty `DataFetcherExceptionHandlerResult` ?

Reply:
Ah yeah I missed that, let's keep it `@NotNull`